### ISSUE TITLE
External Link Manager for Zim as plugin ?

### ISSUE THREAD

**Issue by Nick222** [zim-desktop-wiki/zim-desktop-wiki#537]
Can somebody create link-manager for Zim to view (and - may be - to edit):

a) List of all external links in Zim-notebook - including pictures and files - with types of links and special mark for links without real files or pictures (which was moving or deleting) ?

b) List of files in Zim-files-folder without any links on it in Zim-notebook ?

Now - as I see - user cannot find link in Zim-notebook by link`s file name (only by link`s name)...

**Comment by jaap-karssenberg** [comment]
Can you explain the issue you try to solve with this?  Use case not quite clear to me.

**Comment by Nick222** [comment]
There are a number of problems with using ZIM related to file links.

1) If I changed, transferred or deleted the file to which there is a link in the ZIM note, I cannot remember where this link was (there are a lot of pages in the notebook). Thus, much later I will see that the necessary link to the file is broken, and the file is already changed or deleted.
The situation is even worse if there are links to the same file in different notes.

2) There is a folder specified in the notepad settings, the files in which it is easy to make links.
If I deleted all the links to a file in this folder in Notepad, then I can transfer this file to another place or delete it altogether.
But how to understand - is there links to this file in notepad or not?

So I thought that we needed some kind of "file link manager" to help in such situations ...

(Excuse me - Google Translator)
